### PR TITLE
Add Get method

### PR DESCRIPTION
**Pull request by JakeChampion**
This PR adds the [`Get` abstract operation](http://www.ecma-international.org/ecma-262/6.0/#sec-get-o-p).

### CODE REVIEW

File: es6.js [hunk 1]
```diff
@@ -224,6 +224,23 @@ var ES6 = assign(assign({}, ES5), {
 	// https://people.mozilla.org/~jorendorff/es6-draft.html#sec-samevaluezero
 	SameValueZero: function SameValueZero(x, y) {
 		return (x === y) || ($isNaN(x) && $isNaN(y));
+	},
+
+	/**
+	 * 7.3.1 Get (O, P) - http://www.ecma-international.org/ecma-262/6.0/#sec-get-o-p
+	 * 1. Assert: Type(O) is Object.
```
Comment (ljharb) [inline es6.js]: This assertion means that it should throw a TypeError if it's not an object - instead of just silently nooping.

File: es6.js [hunk 2]
```diff
@@ -224,6 +224,23 @@ var ES6 = assign(assign({}, ES5), {
...
+	 * 2. Assert: IsPropertyKey(P) is true.
+	 * 3. Return O.[[Get]](P, O).
+	 */
+	Get: function Get(O, P) {
+		// 7.3.1.1
+		if (typeof O === 'object') {
+			// 7.3.1.2
+			if (typeof P === 'string' || typeof P === 'symbol') {
```
Comment (ljharb) [inline es6.js]: similarly, this should throw if this condition is not met

**Comment by ljharb** [comment]
@JakeChampion would you mind checking the "allow edits" checkbox on the right column? That way I can handle rebasing it if needed.

**Comment by JakeChampion** [comment]
@ljharb I have checked the "Allow edits from maintainers" checkbox :-)

**Pull request merged**